### ISSUE TITLE
[SUGGESTION] The box that shows what pokemon are in a dungeon should show which are bosses and which appear normally

### ISSUE THREAD

**Issue by wiebejamin** [pokeclicker/pokeclicker#1425]
**Current Behavior**

It just shows them all in a list. The boss pokemon are at the end, but there are often several options for boss pokemon, and the game makes no indication of where the delimiter is.

**Improved Behavior**

There should be some indication, either symbols on the boss pokemon or just a line that shows where the normal list ends and where the bosses start

**Reason for change**

Often times it's unclear whether a pokemon is at the end of a list of standard pokemon, or the start of a list of bosses

**Detailed Description**

Basically it'd be nice if there was any way of knowing just by looking which pokemon on the list are options for the dungeon boss, and which are just found around.

**Screenshots:**

![image](https://user-images.githubusercontent.com/26973120/120131729-d6939100-c17d-11eb-82e7-780be0f864fd.png)
For example, in this image, I can't tell if Litwick is supposed to be a dungeon boss or not

**Comment by Logman4** [comment]
You could just check the wiki. And using your example you could just use this [link](https://pokeclicker.miraheze.org/wiki/Celestial_Tower).

**Comment by wiebejamin** [comment]
I mean, yeah, but by that logic, why should it say anything? Why bother mentioning what pokemon or trainers show up, or if you've caught them? You can just look up everything there and cross reference with your pokedex. It would be convenient, that's why I want it, not because it's necessary.

**Comment by RedSparr0w** [comment]
closed via #1434
